test/filegroup/BUILD

```diff
@@ -12,3 +12,14 @@ plz_e2e_test(
     name = "filegroup_concurrent_build_test",
     cmd = "plz clean //test/filegroup:gen && plz build //test/filegroup/many:all",
 )
+
+hash_filegroup(
+    name = "hashed",
+    srcs = ["hash_filegroup_test.txt"],
```

Comment:
Might be nice to have 2 files here so we trigger that loop in the bash

Reply:
Yeah fair, don't quite have time now though, will revisit later.